README.md

```diff
@@ -1,10 +1,13 @@
 # Atoti UI Migration
 
-A command line interface to migrate content that was saved in older versions of Atoti UI to be compatible with newer versions.
+A command line interface to migrate:
+
+- an Atoti UI application content that was saved in older versions of Atoti UI to be compatible with newer versions.
```

Comment:
In particular, I'd like to avoid _"an"_ at the beginning of the sentence, to make it clearer that this is a generic tool, not dedicated to a specific Atoti project.

```suggestion
- content saved in an old version of Atoti UI, making it compatible with newer versions.
```